ui-tests/jest.config.js

```diff
@@ -1,3 +1,5 @@
+const isHeadless = process.env.HEADLESS !== 'false';
```

Comment:
super nit: I think `!!process.env.HEADLESS` does the same thing

Reply:
Also for strings?

Reply:
<img width="156" alt="image" src="https://github.com/user-attachments/assets/4aa686a9-c2c1-4e6b-81cf-ab0613fee6c3">